pcmanfm/desktopwindow.cpp

```diff
@@ -315,6 +315,15 @@ void DesktopWindow::updateWallpaper() {
           image = loadWallpaperFile(desiredSize); // load the scaled image
         }
       }
+      else if(wallpaperMode_ == WallpaperZoom) {
```

Comment:
Looks like we can just reuse existing code?
`
if(if(wallpaperMode_ == WallpaperFit || wallpaperMode_ == WallpaperZoom) {
    ....
    .... wallpaperMode_ == WallpaperFit ? Qt::KeepAspectRatio : Qt::KeepAspectRatioExpanding);
`